src/lib.rs

```diff
@@ -98,6 +98,27 @@ impl<T> Arena<T> {
     pub fn is_empty(&self) -> bool {
         self.count() == 0
     }
+
+    #[inline]
+    pub fn get(&self, id: NodeId) -> Option<&Node<T>> {
```

Comment:
Please add documentation and a test as well. Why is the inlining necessary here?